malcolm/modules/ADCore/parts/detectordriverpart.py

```diff
@@ -141,7 +151,7 @@ def configure(self,
                   completed_steps,  # type: ACompletedSteps
                   steps_to_do,  # type: AStepsToDo
                   part_info,  # type: APartInfo
-                  generator,  # type: AGenerator
+                  generator,  # type: scanning.util.AGenerator
```

Comment:
I reckon all the other hook annotypes should be namespaced too

Reply:
Sorry, I don't understand this comment.

Reply:
got it.